src/symmetry.jl

```diff
@@ -333,3 +333,9 @@ function unfold_kcoords(kcoords, symmetries)
         normalize_kpoint_coordinate(round.(k; digits))
     end
 end
+
+# Filter the wavevectors that don't have an opposite, to ensure fourier_coeffs is real in
+# real space
+function force_real!(fourier_coeffs, basis)
```

Comment:
Not sure about the name, because could be mistaken as a cast to a real-valued array. Don't have a good alternative in mind though right now.

Also typically we have the basis as first argument in such functions (even though we are not fully consistent).